### ISSUE TITLE
A missing chart type causes a online_design crash

### ISSUE THREAD

**Issue by pbchase** [ctsit/redcap_chart_field#10]
If the module is enabled globally and chart type is never set (i.e. Chart.js vs. Chartist.js), accessing online_designer will cause an error that disables the chart_field module.  

Here is the error:

```
REDCap crashed due to an unexpected PHP fatal error!

Error message: Uncaught Error: Unsupported operand types in
/vagrant/modules/redcap_chart_field_v0.0.0/ExternalModule.php:274 Stack trace:
#0 /vagrant/modules/redcap_chart_field_v0.0.0/ExternalModule.php(64):
REDCapChartField\ExternalModule\ExternalModule->buildConfigFormFields() #1
[internal function]:
REDCapChartField\ExternalModule\ExternalModule->redcap_every_page_top('29') #2
/var/www.backup/redcap-
20180403T1535/redcap_v8.3.0/ExternalModules/classes/ExternalModules.php(1186):
call_user_func_array(Array, Array) #3 /var/www.backup/redcap-
20180403T1535/redcap_v8.3.0/ExternalModules/classes/ExternalModules.php(1253):
ExternalModules\ExternalModules::startHook('redcap_chart_fi...', 'v0.0.0',
Array) #4 /var/www.backup/redcap-
20180403T1535/redcap_v8.3.0/Classes/Hooks.php(37):
ExternalModules\ExternalModules::callHook('every_page_top', Array) #5
/var/www.backup/redcap-
20180403T1535/redcap_v8.3.0/Views/HeaderProject.php(39):
Hooks::call('redcap_every_pa...', Array) #6 /var/www.backup/redcap-
20180403T1535/redcap_v8.3.0/ProjectGe

File: /vagrant/modules/redcap_chart_field_v0.0.0/ExternalModule.php
Line: 274

The "redcap_chart_field" external module was automatically disabled in order
to allow REDCap to function properly. The REDCap administrator has been
notified. Please save a copy of the above error and fix it before re-enabling
the module.
```

This makes enabling the module globally hazardous for all parties. 

I recommend that the module have a default chart type when enabled globally.

**Comment by pbchase** [comment]
Note, this occurs only if the module is enabled _globally_.  That has been clarified above.  A per project-enabling is completely safe.

**Comment by pbchase** [comment]
fixed by PR #11
